src/python/AsyncStageOut/TransferDaemon.py

```diff
@@ -79,6 +79,17 @@ def __init__(self, config):
                 else:
                     self.logger.error('Unknown error in mkdir' % e.errno)
                     raise
+
+       if not os.path.isdir("/tmp/DashboardReport"):
+            try:
+                os.makedirs("/tmp/DashboardReport")
+            except OSError as e:
+                if e.errno == errno.EEXIST:
+                    pass
+                else:
+                    self.logger.error('Unknown error in mkdir' % e.errno)
```

Comment:
Is this branch stopping the TransferDaemon. If not, we can probably use `self.logger.exception` to have the stacktrace and the error message itself (*). It is just to increase the chances to see this in the logs, if it is the last message we are ok, otherwise if the execution continues it a big stacktrace is easier to spot (and more invofmative)!

Anyhow, it is really a minor comment, I will merge this anyhow.

(*)
```python
self.logger.exception('Unknown error in mkdir' % e.errno)
```